example/mqtt-protocol-gateway-example/src/main/java/org/eclipse/hono/example/protocolgateway/Config.java

```diff
@@ -0,0 +1,83 @@
+/*******************************************************************************
+ * Copyright (c) 2020 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+
+package org.eclipse.hono.example.protocolgateway;
```

Comment:
how about making the package name more specific to the purpose, e.g. `org.eclipse.hono.gateway.azure`?